src/ign.in

```diff
@@ -47,7 +47,7 @@ commands = {}
 yaml_found = false
 
 conf_dirs = '@CMAKE_INSTALL_PREFIX@/share/ignition/'
-conf_dirs = ENV['IGN_CONFIG_PATH'] if ENV.key?('IGN_CONFIG_PATH')
+conf_dirs = ENV['GZ_CONFIG_PATH'] if ENV.key?('GZ_CONFIG_PATH')
```

Comment:
We need to tick-tock this variable. 

I think this should work: concatenate the result from `IGN_CONFIG_PATH` in the end of `GZ_CONFIG_PATH`, separated by `ENV_PATH_DELIMITER`.

All other usage can remain as it is, and we should add a note to the migration guide.

Reply:
Hope I did it right? https://github.com/gazebosim/gz-tools/pull/90/commits/aad3b5a06550699214b0a3e22bd1e14dbf6f8661

Reply:
Thanks, that worked for me. I checked that `GZ_CONFIG_PATH` overrides the default install, thus keeping the current behaviour of `IGN_CONFIG_PATH`.

And `IGN_CONFIG_PATH` gets appended to either the default install of `GZ_CONFIG_PATH` according to what's set. That could be considered a slight change in behaviour, because now `IGN_CONFIG_PATH` doesn't override the default install anymore. But I think it's a good way to make sure we support a mix of paths while the user is transitioning.